client/src/pages/index.tsx

```diff
@@ -1,205 +1,315 @@
-          ) : (
-            <Box css={{ height: 52 }} />
-          )}
+          <Dialog open={mobileMenuOpen} onOpenChange={handleCloseMobileMenu}>
+            <StyledDialogContent aria-label="Mobile menu">
+              <SideBarMenuMobile />
+            </StyledDialogContent>
+          </Dialog>
 
-          <Box>
+          <SideBarContainer>
+            <SideBarMenu />
+          </SideBarContainer>
+
+          <Box css={{ pb: '$4', pt: '$8' }}>
             <IconButton
-              onClick={() =>
-                theme === "dark" ? setTheme("light") : setTheme("dark")
-              }
+              css={{
+                '@sm': {
+                  display: 'contents',
+                },
+                '@lg': {
+                  display: 'none',
+                },
+                padding: 0,
+                mb: '$4',
+              }}
             >
-              <SunIcon height={14} width={14} />
+              <HamburgerMenuIcon
+                onClick={() => setMobileMenuOpen(!mobileMenuOpen)}
+              />
             </IconButton>
-          </Box>
-        </Box>
+            <Heading as={'h3'} size={'xl'} css={{ fontWeight: 700, pb: '$8' }}>
+              Overview
+            </Heading>
+            {/*  Overview Chart */}
+            <Stack
+              direction={{
+                '@initial': 'column',
+                '@sm': 'row',
+              }}
+              css={{
+                pt: '$5',
+              }}
+            >
+              <PlaceHolder css={{ borderRadius: '5px 0px 0px 5px' }}>
+                <Box id="number-of-txs" css={{ pt: '$4' }}>
+                  <Text size={'sm'}>Price evolution (last 30 days)</Text>
+                  <Box css={{ position: 'relative', height: 200, mt: '$4' }}>
+                    <Box
+                      css={{
+                        position: 'absolute',
+                        width: '100%',
+                        height: '100%',
+                        left: 0,
+                        top: 0,
+                      }}
+                    >
+                      <NbTxsPerDay statsData={statsData.nbTxsPerDay} />
+                    </Box>
+                  </Box>
+                </Box>
+              </PlaceHolder>
 
-        <Heading as={"h1"} size={"3xl"} css={{ mt: "$8" }}>
-          Get the latest data from Stacks blockchain
-        </Heading>
-        <Heading size={"2xl"} css={{ fontWeight: 400 }}>
-          A project made by{" "}
-          <Link
-            href="https://www.sigle.io/"
-            target="_blank"
-            rel="noreferrer"
-            css={{ fontFamily: "$lato" }}
-          >
-            Sigle
-          </Link>
-        </Heading>
+              <PlaceHolder css={{ borderRadius: '0px 5px 5px 0px' }}>
+                <Stack direction="column" css={{}}>
+                  <Title>STX price</Title>
+                  <TextOverview>
+                    $2,46{' '}
+                    <text
```

Comment:
this is using the HTML text component instead of the Stitches `Text` component. In general, the color should be coming from the theme, using a hardcoded value can create issue while changing themes.